src/model/modifier/__tests__/RichTextEditorUtil-test.js

```diff
@@ -169,3 +174,49 @@ test('tryToRemoveBlockStyleonDelete breaks out of code block on enter two blank
 
   expect(lastBlock.toJS()).toMatchSnapshot();
 });
+
+describe('onTab on list block', () => {
+  const setListBlock = (contentState, type) =>
+    DraftModifier.setBlockType(contentState, selectionState, type);
+  const changeBlockType = setListItem =>
+    EditorState.push(editorState, setListItem, 'change-block-type');
+  const getFirstBlockDepth = contentState =>
+    contentState
+      .getCurrentContent()
+      .getFirstBlock()
+      .getDepth();
+  const addTab = (contentState, maxDepth = 2) =>
+    onTab({preventDefault: () => {}}, contentState, maxDepth);
+
+  test('increases the depth of unordered-list-item', () => {
+    const contentState = editorState.getCurrentContent();
+    const setListItem = setListBlock(contentState, 'unordered-list-item');
+    const withListItem = changeBlockType(setListItem);
+
+    const afterFirstTab = addTab(withListItem);
+    const depthAfterFirstTab = getFirstBlockDepth(afterFirstTab);
+
+    expect(depthAfterFirstTab).toBe(1);
+
+    const afterSecondTab = addTab(afterFirstTab);
+    const depthAfterSecondTab = getFirstBlockDepth(afterSecondTab);
+
+    expect(depthAfterSecondTab).toBe(2);
+  });
+
+  test('increases the depth of unordered-list-item', () => {
```

Comment:
nit, should be: increases the depth of **ordered**-list-item

Reply:
Yeah, I should... :(

copy/paste miss...